src/tigerbeetle-java/src/main/java/com/tigerbeetle/Accounts.java

```diff
@@ -0,0 +1,317 @@
+package com.tigerbeetle;
+
+import java.nio.ByteBuffer;
+
+/**
+ * A {@link Batch batch} of accounts.
+ */
+public final class Accounts extends Batch {
+
+    interface Struct {
+
+        public static final int Id = 0;
+        public static final int UserData = 16;
+        public static final int Reserved = 32;
+        public static final int Ledger = 80;
+        public static final int Code = 84;
+        public static final int Flags = 86;
+        public static final int DebitsPending = 88;
+        public static final int DebitsPosted = 96;
+        public static final int CreditsPending = 104;
+        public static final int CreditsPosted = 112;
+        public static final int Timestamp = 120;
+
+        public static final int SIZE = 128;
+    }
+
+    static final Accounts EMPTY = new Accounts(0);
+
+    /**
+     * Constructs an empty batch of accounts with the desired maximum capacity.
+     * <p>
+     * Once created, an instance cannot be resized, however it may contain any number of accounts
+     * between zero and its {@link #getCapacity capacity}.
+     *
+     * @param capacity the maximum capacity.
+     *
+     * @throws IllegalArgumentException if capacity is negative.
+     */
+    public Accounts(final int capacity) {
+        super(capacity, Struct.SIZE);
+    }
+
+    Accounts(final ByteBuffer buffer) {
+        super(buffer, Struct.SIZE);
+    }
+
+    /**
+     * Adds a new account at the end of this batch.
+     * <p>
+     * If successfully, moves the current {@link #setPosition position} to the newly created
+     * account.
+     *
+     * @throws IllegalStateException if this batch is read-only.
+     * @throws IndexOutOfBoundsException if exceeds the batch's capacity.
+     */
+    @Override
+    public void add() {
+        super.add();
+    }
+
+    /**
+     * Gets a unique, client-defined identifier for the account.
```

Comment:
For these id getters could you say "Gets the unique" instead of "Gets a unique" because the latter makes it sound slightly as if it's random or something.